backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/output/database/service/DBService.java

```diff
@@ -154,7 +156,19 @@ public void deleteAnnouncementByNormEli(DeleteAnnouncementByNormEliPort.Command
 
   @Override
   @Transactional
-  public void deleteNorm(DeleteNormPort.Command command) {
-    normRepository.deleteByEliManifestation(command.eli().toString());
+  public void deleteUnpublishedNorm(DeleteUnpublishedNormPort.Command command) {
```

Comment:
Why having two different output ports for the process of deleting norms by an eli (work or manifestation) and a publish state? Why not just haven 1 output port for both specific use cases. This port would take:
1. Eli (work or manifestation), which delete query to call can be decided by the eli right?
2. Publish state
Then the application service containing the one output port would decide by which eli to delete and on which publish state and the DB service using the only port to decide which sql query to run

Reply:
oh yes they now use the same logic. I'll combine them :)

Reply:
nevermind one of those ports works on work level elis and the other one on manifestation elis so we can't combine them that trivially. So I'll keep it as is for now. (Otherwise we should probably be able to handle all combinations of elis and publish-states and I'm not sure we should do that)